hockeysdk/src/main/java/net/hockeyapp/android/objects/CrashDetails.java

```diff
@@ -96,12 +102,15 @@ public CrashDetails(String crashIdentifier, Throwable throwable, String managedE
             //exception, The throwable will be the Java exception.
             if (!TextUtils.isEmpty(managedExceptionString)) {
                 //Print the java exception
-                throwable.printStackTrace(printWriter);
+                PrintWriter javaExceptionWriter = new BoundedPrintWriter(stackTraceResult, CRASH_FILE_STACKTRACE_MAX_SIZE / 2);
```

Comment:
`new BoundedPrintWriter(`~stackTraceResult~ **printWriter**
Currently, the counter in `printWriter` is not used properly